### ISSUE TITLE
Please provide your Apple Developer Program account credentials

### ISSUE THREAD

**Issue by philCc** [fastlane/fastlane#17152]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.15.5                                     |
| Ruby                        | 2.6.5                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.24.3 (Apple Git-128)          |
| Installation Source         | ~/.rvm/gems/ruby-2.6.5/bin/fastlane         |
| Host                        | Mac OS X 10.15.5 (19F101)                   |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.6.5/lib                |
| OpenSSL Version             | OpenSSL 1.1.1g  21 Apr 2020                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 11.6                                        |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile.swift`</summary>

```ruby
// This file contains the fastlane.tools configuration
// You can find the documentation at https://docs.fastlane.tools
//
// For a list of all available actions, check out
//
//     https://docs.fastlane.tools/actions
//

import Foundation

class Fastfile: LaneFile {
	override init() {
		super.init()
	}

	func pipeLane(withOptions options:[String: String]?) {
		updateLane(withOptions: options)
		ipaLane(withOptions: options)
		deployLane(withOptions: options)
		commitLane(withOptions: options)
		maketagLane(withOptions: options)
		sentryLane(withOptions: options)
	}

	func updateLane(withOptions options:[String: String]?) {
		sh(command: "pod install")
		sh(command: "carthage update --platform iOS")
	}

	func ipaLane(withOptions options:[String: String]?) {
		let env = Env.instance(by: options)
		incrementBuildNumber(xcodeproj: xcodeproj)
		match(
			type: env.sign,
			readonly: false,
			appIdentifier: [env.identifier],
			teamId: teamID,
			forceForNewDevices: true
		)
		updateCodeSigningSettings(
			path: xcodeproj,
			teamId: teamID,
			codeSignIdentity: "iPhone Distribution",
			profileName: "${sigh_\(env.identifier)_\(env.sign)}"
		)
		buildApp(
			workspace: workspace, 
			scheme: Config.scheme,
			clean: true,
			outputDirectory: "./output",
			configuration: env.config,
			exportMethod: env.export,
            xcargs: "FLUTTER_BUILD_MODE=release"
		) 
		automaticCodeSigning(
			path: xcodeproj,
			useAutomaticSigning: true,
			codeSignIdentity: "iPhone Developer"
		)
	}

	func deployLane(withOptions options:[String: String]?) {
		let env = Env.instance(by: options)
		guard env.export != ExportType.appstore.rawValue else {
			deliver(
				appVersion: env.version,
				ipa: "./output/\(Config.scheme).ipa",
				skipBinaryUpload: false,
				skipScreenshots: true,
				skipMetadata: true,
				force: true,
				submitForReview: false,
				automaticRelease: false,
                precheckIncludeInAppPurchases: false
			)
			return
		}
		var changelog = ""
			if !env.commitId.isEmpty {
				changelog = changelogFromGitCommits(
					between: ["\(env.commitId)", "HEAD"],
					pretty: "- (%an) %s"
				)
			}
		plugin("fir_cli", args: [
			"api_token": fir_cli_api_token,
			"specify_file_path": "./output/\(Config.scheme).ipa",
			"changelog": changelog,
			"dingtalk_access_token": dingtalk_access_token,
			"dingtalk_custom_message": "\n\(env.config)版本上传成功",
			"dingtalk_at_phones": dingtalk_at_phones
			])
	}

	func commitLane(withOptions options:[String: String]?) {
		let env = Env.instance(by: options)
		gitPull()
		gitAdd(path: "./")
		gitCommit(path: "./", message: "- [fastlane] \(env.version)(Build \(env.buildNum))")
		pushToGitRemote()
	}

	func maketagLane(withOptions options:[String: String]?) {
		let env = Env.instance(by: options)
		let tagName = "\(Config.scheme)/\(env.version)/\(env.sign)-\(env.buildNum)"
		sh(command: "git tag -d $(git tag -l | grep \(tagName))")
		sh(command: "git push origin :refs/tags/\(tagName)")
		addGitTag(
			tag: tagName, 
			grouping: env.config,
			buildNumber: env.buildNum,
			message: "- [fastlane] auto make tag"
		)
		sh(command: "git push --tags origin")
	}

	func sentryLane(withOptions options:[String: String]?) {
		let env = Env.instance(by: options)
		var projectSlug = ProjectSlug.prod.rawValue
		if env.export != ExportType.appstore.rawValue {
			projectSlug = ProjectSlug.dev.rawValue
		}
		plugin("sentry_upload_dsym", args: [
			"auth_token": SENTRY_TOKEN,
			"org_slug": "sentry",
			"project_slug": projectSlug,
			"dsym_path": "./output/\(Config.scheme).app.dSYM.zip"
		])
	}

	private func plugin(_ name: String, args: [String: Any]?) -> String {
		let _args = args?.map({ RubyCommand.Argument(name: $0.key, value: $0.value) }) ?? []
		let command = RubyCommand(commandID: "", methodName: name, className: nil, args: _args)
		return runner.executeCommand(command)
	}
}

```
</details>

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.157.3 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                             | Version | Update-Status |
| ---------------------------------- | ------- | ------------- |
| fastlane-plugin-pgyer              | 0.2.2   | ✅ Up-To-Date  |
| fastlane-plugin-sentry             | 1.6.0   | ✅ Up-To-Date  |
| fastlane-plugin-fir_cli            | 2.0.9   | ✅ Up-To-Date  |
| fastlane-plugin-ding_talk          | 0.1.1   | ✅ Up-To-Date  |
| fastlane-plugin-ding_talk_msg_push | 0.1.2   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                | Version      |
| ---------------------------------- | ------------ |
| did_you_mean                       | 1.3.0        |
| executable-hooks                   | 1.6.0        |
| bundler-unload                     | 1.0.2        |
| rubygems-bundler                   | 1.4.5        |
| bundler                            | 2.1.4        |
| slack-notifier                     | 2.3.2        |
| rouge                              | 2.0.7        |
| xcpretty                           | 0.3.0        |
| terminal-notifier                  | 2.0.0        |
| unicode-display_width              | 1.7.0        |
| terminal-table                     | 1.8.0        |
| plist                              | 3.5.0        |
| CFPropertyList                     | 3.0.2        |
| addressable                        | 2.7.0        |
| multipart-post                     | 2.0.0        |
| word_wrap                          | 1.0.0        |
| tty-cursor                         | 0.7.1        |
| tty-spinner                        | 0.9.3        |
| babosa                             | 1.0.3        |
| colored                            | 1.2          |
| highline                           | 1.7.10       |
| commander-fastlane                 | 4.4.6        |
| faraday                            | 1.0.1        |
| faraday_middleware                 | 1.0.0        |
| gh_inspector                       | 1.1.3        |
| mini_magick                        | 4.10.1       |
| rubyzip                            | 2.3.0        |
| security                           | 0.1.3        |
| xcpretty-travis-formatter          | 1.0.0        |
| naturally                          | 2.2.0        |
| simctl                             | 1.6.8        |
| uber                               | 0.1.0        |
| declarative                        | 0.0.20       |
| declarative-option                 | 0.1.0        |
| representable                      | 3.0.4        |
| retriable                          | 3.1.2        |
| mini_mime                          | 1.0.2        |
| signet                             | 0.14.0       |
| httpclient                         | 2.8.3        |
| google-api-client                  | 0.38.0       |
| emoji_regex                        | 3.0.0        |
| nanaimo                            | 0.3.0        |
| colored2                           | 3.1.2        |
| claide                             | 1.0.3        |
| atomos                             | 0.1.3        |
| xcodeproj                          | 1.18.0       |
| public_suffix                      | 4.0.6        |
| tty-screen                         | 0.8.1        |
| excon                              | 0.76.0       |
| unf_ext                            | 0.0.7.7      |
| unf                                | 0.1.4        |
| domain_name                        | 0.5.20190701 |
| http-cookie                        | 1.0.3        |
| faraday-cookie_jar                 | 0.0.7        |
| fastimage                          | 2.2.0        |
| json                               | 2.3.1        |
| dotenv                             | 2.7.6        |
| jwt                                | 2.2.2        |
| multi_json                         | 1.15.0       |
| os                                 | 1.1.1        |
| memoist                            | 0.16.2       |
| googleauth                         | 0.13.1       |
| rake                               | 13.0.1       |
| digest-crc                         | 0.6.1        |
| google-cloud-errors                | 1.0.1        |
| google-cloud-env                   | 1.3.3        |
| google-cloud-core                  | 1.5.0        |
| google-cloud-storage               | 1.28.0       |
| aws-eventstream                    | 1.1.0        |
| aws-sigv4                          | 1.2.2        |
| aws-partitions                     | 1.364.0      |
| jmespath                           | 1.4.0        |
| aws-sdk-core                       | 3.105.0      |
| aws-sdk-kms                        | 1.37.0       |
| aws-sdk-s3                         | 1.79.1       |
| forwardable                        | 1.2.0        |
| logger                             | 1.3.0        |
| stringio                           | 0.0.2        |
| ipaddr                             | 1.2.2        |
| openssl                            | 2.1.2        |
| ostruct                            | 0.1.0        |
| strscan                            | 1.0.0        |
| date                               | 2.0.0        |
| fileutils                          | 1.1.0        |
| etc                                | 1.0.1        |
| io-console                         | 0.4.7        |
| zlib                               | 1.0.0        |
| rexml                              | 3.1.9        |
| psych                              | 3.1.0        |
| mutex_m                            | 0.1.0        |
| webrick                            | 1.4.2        |
| fastlane-plugin-pgyer              | 0.2.2        |
| fastlane-plugin-sentry             | 1.6.0        |
| http-accept                        | 1.7.0        |
| mime-types-data                    | 3.2020.0512  |
| mime-types                         | 3.3.1        |
| netrc                              | 0.11.0       |
| rest-client                        | 2.1.0        |
| api_tools                          | 0.1.0        |
| thor                               | 0.20.3       |
| admqr_knife                        | 0.1.5        |
| ruby_android_apk                   | 0.7.7.1      |
| chunky_png                         | 1.3.12       |
| rqrcode                            | 0.10.1       |
| fir-cli                            | 2.0.9        |
| fastlane-plugin-fir_cli            | 2.0.9        |
| fastlane-plugin-ding_talk          | 0.1.1        |
| fastlane-plugin-ding_talk_msg_push | 0.1.2        |
</details>

*generated on:* **2020-09-04**
</details>

![WX20200904-133642](https://user-images.githubusercontent.com/626401/92203457-c8953600-eeb3-11ea-8ecb-40e943438beb.png)

**Comment by joshdholtz** [comment]
Duplicate of #17153. Working on now. Sorry about issue 😔

**Comment by joshdholtz** [comment]
This should be fixed in `2.157.4` which was just released!
